src/main/java/cow/commands/EventCommand.java

```diff
@@ -0,0 +1,49 @@
+package cow.commands;
+
+import cow.exceptions.CowExceptions;
+import cow.filesaver.FileSaver;
+import cow.message.Ui;
+import cow.tasks.Event;
+import cow.tasks.Task;
+import cow.todolist.TodoList;
+
+
+// solution below inspired by https://github.com/se-edu/addressbook-level2/tree/master
+/** Creates an Event command object. **/
+public class EventCommand extends Command {
+    public static final String COMMAND_WORD = "event";
+    public static final String COMMAND_EXAMPLE = "event project meeting /from Mon 2pm /to 4pm";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an Event item to the list.\n"
+            + "Example: " + COMMAND_EXAMPLE;
+
+    private final String from;
+    private final String to;
+    private final String description;
+
+    /**
+     * Creates an EventCommand instance.
+     * @param description of the event.
+     * @param from Start of Event.
+     * @param to End of event.
```

Comment:
This comment applies for all your Javadoc comments. There should be an empty line between the description and parameter section.